frontend_egui/src/app.rs

```diff
@@ -567,6 +569,15 @@ impl SnowGui {
                         }
                     }
                 });
+                ui.menu_button("Scaling algorithm", |ui| {
+                    ui.set_min_width(Self::SUBMENU_WIDTH);
+                    for algorithm in Self::SCALING_ALGORITHMS {
+                        if ui.button(format!("{}", algorithm)).clicked() {
```

Comment:
You can use `ui.radio_value` to show radio buttons so it is visible what is currently selected: https://docs.rs/egui/0.30.0/egui/struct.Ui.html#method.radio_value